flight/Modules/Stabilization/stabilization.c

```diff
@@ -520,26 +525,35 @@ static void stabilizationTask(void* parameters)
 
 		actuatorDesired.Thrust = stabDesired.Thrust;
 
-		static uint32_t last_pos_thrust_time = 0;
+		static uint32_t last_nonzero_thrust_time = 0;
+		static bool last_thrust_pos = true;
 
-		if (stabDesired.Thrust > THROTTLE_EPSILON) {
+		if (flightStatus.Armed != FLIGHTSTATUS_ARMED_ARMED) {
+			actuatorDesired.Thrust = 0.0f;
+			last_thrust_pos = true;
+			last_nonzero_thrust_time = 0;
+		} else if (fabsf(stabDesired.Thrust) > THROTTLE_EPSILON) {
 			if (settings.LowPowerStabilizationMaxTime) {
-				last_pos_thrust_time = this_systime;
+				last_nonzero_thrust_time = this_systime;
+				last_thrust_pos = stabDesired.Thrust >= 0;
 			}
-		} else if (last_pos_thrust_time) {
-			if ((this_systime - last_pos_thrust_time) <
+		} else if (last_nonzero_thrust_time) {
+			if ((this_systime - last_nonzero_thrust_time) <
 					1000.0f * settings.LowPowerStabilizationMaxTime) {
-				/* Choose a barely-positive value to trigger
+				/* Choose a barely-nonzero value to trigger
 				 * low-power stabilization in actuator.
 				 */
-				actuatorDesired.Thrust = THROTTLE_EPSILON;
+				if (last_thrust_pos) {
```

Comment:
The handling of last_thrust_pos is OK but I still occasionally am surprised to see motor thrust reversals on the ground.  It may be possible to improve this.  (Doesn't really -matter-)